### PR TITLE
Feat/alpha day01

### PR DESCRIPTION
**Pull request by kayac-chang**

### CODE REVIEW

**Comment by alphatero** [comment]
求救 如何刪除不應該出現在這個分支的day2檔案

File: day01/alpha/main.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+const getKeycode = (e) => {
+  return e.key.toUpperCase().charCodeAt();
+};
+
+const play = (currentAudio) => {
+  //if keydown is not in keys then return
+  if (!currentAudio) return;
+  //initial audio time;
+  currentAudio.currentTime = 0;
+  //play() is audio tag's function;
```
Comment (kayac-chang) [inline day01/alpha/main.js:14]: remove unnecessary comment

File: day01/alpha/main.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  currentAudio.play();
+};
+
+const toggleClass = (currentKey, isActive = false) => {
+  if (!currentKey) return;
+  currentKey.classList.toggle("play", isActive);
+};
+
+const keyDown = (e) => {
+  const currentAudio = document.querySelector(
```
Comment (kayac-chang) [inline day01/alpha/main.js:24]: don't have to query element every time when event fired

File: day01/alpha/src/header.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const header = document.querySelector("header");
+
+header.innerHTML = `<nav class="bg-white bg-opacity-75 fixed top-0 w-full">
```
Comment (kayac-chang) [inline day01/alpha/src/header.js:3]: this format can be improved

File: day01/alpha/src/header.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+<ul class="flex justify-between p-4">
+  <li><h1 class="text-2xl">alpha's js 30 days</h1></li>
+  <li><a class="hover:text-blue-500 text-xl" href="${
+    import.meta.env.VITE_APP_GITHUB
```
Comment (kayac-chang) [inline day01/alpha/src/header.js:7]: make a config for env variable

File: day02/alpha/main.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
+import "./style.css";
+import "./src/header";
+
+/**
+ * 利用js來讓指針動的跟真的時鐘一樣
+ */
+
+//select DOM
+const select = (item) => document.querySelector(item);
```
Comment (kayac-chang) [inline day02/alpha/main.js:9]: why day2 existed

**Review by kayac-chang — COMMENTED** [review]
day02 should not be here

File: day02/alpha/main.js (hunk 5 above)
Comment (alphatero) [reply day02/alpha/main.js:9]: 我想重開這隻分支了orz

**Pull request closed without merging**